README.md

```diff
@@ -183,214 +169,61 @@ require('telescope').setup{
-| `file_ignore_patterns` | Pattern to be ignored `{ "scratch/.*", "%.env" }`     | dict                     |
-
-### Customize Default Builtin behavior
-
-You can customize each default builtin behavior by adding the preferred options
-into the table that is passed into `require("telescope").setup()`.
-
-Example:
-
-```lua
-require("telescope").setup {
-  defaults = {
-    -- Your defaults config goes in here
-  },
-  pickers = {
-    -- Your special builtin config goes in here
-    buffers = {
-      sort_lastused = true,
-      theme = "dropdown",
-      previewer = false,
-      mappings = {
-        i = {
-          ["<c-d>"] = require("telescope.actions").delete_buffer,
-          -- Right hand side can also be the name of the action as a string
-          ["<c-d>"] = "delete_buffer",
-        },
-        n = {
-          ["<c-d>"] = require("telescope.actions").delete_buffer,
-        }
-      }
-    },
-    find_files = {
-      theme = "dropdown"
-    }
-  },
-  extensions = {
-    -- Your extension config goes in here
-  }
-}
-```
-
-## Mappings
+## Default Mappings
 
 Mappings are fully customizable.
 Many familiar mapping patterns are setup as defaults.
 
-| Mappings       | Action                                                       |
-|----------------|--------------------------------------------------------------|
-| `<C-n>/<Down>` | Next item                                                    |
-| `<C-p>/<Up>`   | Previous item                                                |
-| `j/k`          | Next/previous (in normal mode)                               |
-| `<cr>`         | Confirm selection                                            |
-| `<C-q>`        | Confirm selection and open quickfix window                   |
-| `<C-x>`        | Go to file selection as a split                              |
-| `<C-v>`        | Go to file selection as a vsplit                             |
-| `<C-t>`        | Go to a file in a new tab                                    |
-| `<C-u>`        | Scroll up in preview window                                  |
-| `<C-d>`        | Scroll down in preview window                                |
+| Mappings       | Action                                      |
+|----------------|---------------------------------------------|
+| `<C-n>/<Down>` | Next item                                   |
+| `<C-p>/<Up>`   | Previous item                               |
+| `j/k`          | Next/previous (in normal mode)              |
+| `H/M/L`        | Select High/Middle/Low (in normal mode)     |
+| `<cr>`         | Confirm selection                           |
+| `<C-x>`        | go to file selection as a split             |
+| `<C-v>`        | go to file selection as a vsplit            |
+| `<C-t>`        | go to a file in a new tab                   |
+| `<C-u>`        | scroll up in preview window                 |
```

Comment:
What are the planned keybindings to scroll the solution/result window?

Reply:
??? `next` and `previous` item i guess. If #42 is resolved. You will just be able to scroll past your current "viewport"

I dont get the comment